src/views/settings/settings-view.jsx

```diff
@@ -89,6 +89,31 @@ export default function SettingsView() {
 		() => Object.keys(settingsToUpdate).length === 0 && disabled,
 		[settingsToUpdate, disabled]
 	);
+	const setNewOrForwardSignatureId = (ItemsAdd, resp, oldSignatureId, isFowardSignature) => {
+		const newOrForwardSignatureToSet = ItemsAdd.find((item) => item.id === oldSignatureId);
+		if (
```

Comment:
You can simplify the entire condition using optional chaining:
if (newOrForwardSignatureToSet && resp?.payload?.response?.Body?.BatchResponse?.CreateSignatureResponse)